index.html

```diff
@@ -367,8 +367,10 @@ <h4 data-i18n="index.page.requirements.updates.header">Frequent Updates</h4>
 </script>
 
 <script>
-    var ext = "";
-    var OS = "UNKNOWN";
+    // Detect Operating System and required file extension
+    var ext = "",
+        OS = "UNKNOWN";
+
     if (/Windows|Win32|WOW64|Win64/.test(navigator.userAgent)) {
```

Comment:
There _shouldn't_ be any changes to UserAgent strings, at least not in common browsers, but good point about the case-insensitivity. I thought that too a while ago.